lib/settings.js

```diff
@@ -157,12 +157,12 @@ async function setReduceMotion (sim, reduceMotion = true) {
   }
 }
 
-async function setDarkMode (sim, darkMode = true) {
-  log.debug(`Setting dark mode ${darkMode ? 'on' : 'off'}`);
+async function setDarkMode (sim, isEnabled = true) {
+  log.debug(`Setting dark mode ${isEnabled ? 'on' : 'off'}`);
   const paths = await plistPaths(sim, 'uiStyleSettings');
   for (const file of paths) {
     await update(file, {
-      UserInterfaceStyleMode: darkMode ? 2 : 1,
+      UserInterfaceStyleMode: isEnabled ? 2 : 1,
```

Comment:
If these are set by integer than it means Apple could add more style modes in the future. Do we still want to keep the current approach or make it more universal?

Reply:
I'd rather keep it as it is and deal with Apple's additions later, than try to predict what they might do.